### PR TITLE
nodejs-jquery-ujs 1.2.1

### PR DESCRIPTION
**Pull request by dLobatog**
Made for https://github.com/theforeman/foreman/pull/3433 - however let's sort out https://github.com/theforeman/foreman/pull/3433 first, once that one is merged I'll rebase. Comps are going to go out of date when the other PR is merged.

### CODE REVIEW

File: nodejs-jquery-ujs/nodejs-jquery-ujs.spec [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Name: nodejs-%{npm_name}
+Version: 1.2.1
+Release: 1%{?dist}
+Summary: Ruby on Rails unobtrusive scripting adapter for jQuery, for npm
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/shakacode/jquery-ujs.git
+Source0: http://registry.npmjs.org/jquery-ujs/-/jquery-ujs-1.2.1.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
```
Comment (domcleal) [inline nodejs-jquery-ujs/nodejs-jquery-ujs.spec]: Not needed, EL5-era.

File: nodejs-jquery-ujs/nodejs-jquery-ujs.spec [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+%setup -q -n package
+
+%build
+#nothing to do
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr script src test MIT-LICENSE *.json *.md %{buildroot}%{nodejs_sitelib}/%{npm_name}
+
+%clean
```
Comment (domcleal) [inline nodejs-jquery-ujs/nodejs-jquery-ujs.spec]: Not required, EL5-era.

File: nodejs-jquery-ujs/nodejs-jquery-ujs.spec [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+%{nodejs_sitelib}/%{npm_name}
+%exclude %{nodejs_sitelib}/%{npm_name}/script
+%exclude %{nodejs_sitelib}/%{npm_name}/*.json
+%exclude %{nodejs_sitelib}/%{npm_name}/*.md
+%exclude %{nodejs_sitelib}/%{npm_name}/.*
+
+%files doc
+%doc %{nodejs_sitelib}/%{npm_name}/README.md
+%doc %{nodejs_sitelib}/%{npm_name}/CONTRIBUTING.md
+%doc %{nodejs_sitelib}/%{npm_name}/MIT-LICENSE
```
Comment (domcleal) [inline nodejs-jquery-ujs/nodejs-jquery-ujs.spec]: Should be in the main subpackage's %files list instead, so it's always installed.

**Comment by dLobatog** [comment]
Rebased

File: comps/comps-foreman-rhel7.xml [hunk 4]
```diff
@@ -76,7 +76,6 @@
       <packagereq type="default">tfm-rubygem-fog-openstack</packagereq>
       <packagereq type="default">tfm-rubygem-fog-rackspace</packagereq>
       <packagereq type="default">tfm-rubygem-fog-vsphere</packagereq>
-      <packagereq type="default">tfm-rubygem-fog-xenserver</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-rhel7.xml]: Unintentional?

File: comps/comps-foreman-rhel7.xml [hunk 5]
```diff
@@ -254,7 +254,6 @@
       <packagereq type="default">tfm-rubygem-fog-openstack-doc</packagereq>
       <packagereq type="default">tfm-rubygem-fog-rackspace-doc</packagereq>
       <packagereq type="default">tfm-rubygem-fog-vsphere-doc</packagereq>
-      <packagereq type="default">tfm-rubygem-fog-xenserver-doc</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-rhel7.xml]: ditto?

**Comment by dLobatog** [comment]
Updated without removing xenserver from comps (not sure how that happened)

**Comment by domcleal** [comment]
Merged as e7eacbf, thanks @dLobatog.

**Pull request closed without merging**